File: frontend/src/types/testimoni.ts
export interface Testimoni {
  id: number;
  nama: string;
  rating: number;
  isi: string;
  foto_url: string | null;
}

export interface TestimoniInput {
  nama: string;
  rating: number;
  isi: string;
  foto_url?: string | null;
}

File: backend/app/models/testimoni.py
from sqlalchemy import Column, Integer, String
from app.database import Base


class Testimoni(Base):
    __tablename__ = "testimonis"

    id = Column(Integer, primary_key=True, index=True)
    nama = Column(String, nullable=False)
    rating = Column(Integer, nullable=False)
    isi = Column(String, nullable=False)
    foto_url = Column(String, nullable=True)